CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ changes.
 
 - `hydra-node` logs will now report `NetworkEvents` to distinguish between `ConnectivityEvent`s and `ReceivedMessage`s on the network.
 
+- Fix the fee calculation for multiple key witnesses in the `hydra-node` internal wallet.
```

Comment:
Could: drop this changelog entry

I'd say this fix is only for the `/commit` endpoint changes we introduced in this version. Hence, it's not really a fix of something that is released.